_sass/app.scss

```diff
@@ -1150,7 +1150,7 @@ a:focus{
 // Content can be presented without loss of information or functionality,
 // and without requiring scrolling in two dimensions for:
 //  - Vertical scrolling content at a width equivalent to 320 CSS pixels
-@media screen and (max-width: 320px) {
+@media screen and (max-width: 319px) {
```

Comment:
I updated the pull request to fix this! It was mainly to avoid the changes introduced to make the page work on zoom from impacting on how it appeared on mobile screens that were 320px, but I've made some changes to make it work better.